low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -119,16 +120,14 @@ def read_stored_cpu_info(self):
             if self.stored_cpu_info is not None and self._node_id in self.stored_cpu_info.keys():
                 self.prev_cpu_info = self.stored_cpu_info[self._node_id]['CPU_LIST']
         except Exception as e:
-            logger.error(f"Error while reading stored cpu info, shutting down CPUFaultSensor : {e}")
-            self.shutdown()
+            raise Exception(f"Error while reading stored cpu info, {e}")
 
     def read_current_cpu_info(self):
         """Read current cpu info"""
         try:
             self.current_cpu_info = self._utility_instance.get_cpu_info()
         except Exception as e:
-            logger.error(f"Error while reading current cpu info, shutting down CPUFaultSensor : {e}")
-            self.shutdown()
+            raise Exception(f"Error while reading current cpu info, {e}")
```

Comment:
this won't help us to find where the actual exception happened. pls use
`raise Exception(f"Error while reading current cpu info") from e`

Reply:
Exception raised from here will be captured and logged in thread_controller.py
`logger.exception(error_msg)` --> This gives full traceback.

Reply:
```
Aug  6 10:10:01 localhost sspl-ll[8262]: ERRORUnrecoverable. IEMSensor has encountered an error: [Errno 2] No such file or directory: '/tmp/test_iem_log_messages'.  (thread_controller.py:126)#012Traceback (most recent call last):#012  File "/opt/seagate/cortx/sspl/low-level/framework/messaging/thread_controller.py", line 119, in execute_thread#012    module.start_thread(conf_reader, msgQlist, product)#012  File "/opt/seagate/cortx/sspl/low-level/framework/base/module_thread.py", line 268, in start_thread#012    self.start()#012  File "/opt/seagate/cortx/sspl/low-level/framework/base/module_thread.py", line 87, in start#012    self._scheduler.run()#012  File "/usr/lib64/python3.6/sched.py", line 154, in run#012    action(*argument, **kwargs)#012  File "/opt/seagate/cortx/sspl/low-level/sensors/impl/generic/iem_sensor.py", line 145, in run#012    self._iem_logs = open(self._log_file_path)#012FileNotFoundError: [Errno 2] No such file or directory: '/tmp/test_iem_log_messages'
```
FYI, @thavanathan